Gruntfile.js

```diff
@@ -427,6 +426,14 @@ module.exports = (grunt) => {
             await buildServiceWorkers.run()
           } else {
             await buildMain.run()
+            if (['.css', '.sass', '.scss'].includes(extension)) {
+              // Esbuild outputs `main.css` and `main.css.map` along `main.js`,
+              // but `index.html` wants to load them from `dist/assets/css`.
+              await fs.promises.copyFile(`${distJS}/main.css`, `${distCSS}/main.css`)
+              if (buildMain.options.sourcemap) {
+                await fs.promises.copyFile(`${distJS}/main.css.map`, `${distCSS}/main.css.map`)
+              }
```

Comment:
Indeed, and it also introduces a bit of code duplication since the same copying operation occurs on the first build.

So I've considered specifying this post-build operation as a custom option, then thought it was better to keep things simple for now; but I should be able to add it if you don't mind the extra option.

I it is also possible via `grunt copy`, but I think it would still be as much specific to esbuild as plain code, and likely a bit slower. I'm also slightly reluctant to increase our dependency to Grunt since we might want to switch to Snowpack later.

Reply:
I think so since this copying operation might not be needed with other build tools, or they might require doing something else.